.pre-commit-config.yaml

```
@@ -0,0 +1,25 @@
+repos:
+  - repo: https://github.com/python-poetry/poetry
+    rev: "1.8.2"
+    hooks:
+      - id: poetry-check
+      - id: poetry-lock
+        args:
+          - --no-update
```

Might be worth disabling the cache here, to avoid inconsistencies locally.
```suggestion
          - --no-update
          - --no-cache
```